### PR TITLE
Populate Sonatype credentials

### PR DESCRIPTION
**Pull request by chanseokoh**

### CODE REVIEW

File: minikube-maven-plugin/kokoro/release_publish.sh [hunk 1]
```diff
@@ -3,6 +3,8 @@
 # Fail on any error.
 set -e
 
+SONATYPE_USERNAME=$(cat "${KOKORO_KEYSTORE_DIR}/72743_sonatype_username")
```
Comment (loosebazooka) [inline minikube-maven-plugin/kokoro/release_publish.sh]: I'm not sure I understand the implications of this.

Comment (chanseokoh) [reply minikube-maven-plugin/kokoro/release_publish.sh]: We were injecting these env variables after externally populating the content of `72743_sonatype_username`, but this external injection is unnecessary. The file `72743_sonatype_username` is already present and the script can just query the content like this.

Comment (chanseokoh) [reply minikube-maven-plugin/kokoro/release_publish.sh]: And then we can remove the setup of the external unnecessary env var injection.

Comment (loosebazooka) [reply minikube-maven-plugin/kokoro/release_publish.sh]: I think @suztomo is reworking this whole release process, no?

Comment (chanseokoh) [reply minikube-maven-plugin/kokoro/release_publish.sh]: Yeah, I'm in touch with @suztomo about this.

**Pull request merged**